pkg/features/embedded/elemental-rootfs/usr/lib/dracut/modules.d/30elemental-rootfs/module-setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+    require_binaries "$systemdutildir"/systemd || return 1
+    return 255
+}
+
+# called by dracut 
+depends() {
+    echo systemd rootfs-block dm fs-lib
+    return 0
+}
+
+# called by dracut
+installkernel() {
+    instmods overlay
+}
+
+# called by dracut
+install() {
+    declare moddir=${moddir}
+    declare systemdutildir=${systemdutildir}
+    declare systemdsystemunitdir=${systemdsystemunitdir}
+
+    inst_multiple \
+        mount mountpoint sort rmdir findmnt rsync cut realpath basename lsblk
+
+    inst_simple "/etc/elemental/config.yaml"
+
+    # Include utilities required for elemental-setup services,
+    # probably a devoted dracut module makes sense
+    inst_multiple -o \
```

Comment:
I think this list needs to be revisited, most of this tools are not needed at all. It should only include the tools required for the elemental-rootfs.service, probably none.